src/controls/OfficeControl.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
```

Comment:
For the files that were renamed, I used `git mv` so that their history would be continuous, but that operation doesn't always succeed.